Project 1/project1/application.py

```diff
@@ -38,20 +38,26 @@ def index():
 def signup():
     if request.method == "POST":
         data = request.form
-        first_name = data.get("first_name")
-        last_name = data.get("email_id")
+        first_name = data.get("fname")
+        last_name = data.get("lname")
+        email_id= data.get("email")
         password = data.get("password")
-        u = Users(first_name, last_name, email_id, password)
+        u = Users(first_name, last_name, email_id, password,)
```

Comment:
You have timestamp in Users model.Are you not storing it?

Reply:
I am storing mam. When creating the User abject itself in the constructor I have created and stored.

Reply:
ok